CONTRIBUTING.md

```diff
@@ -0,0 +1,191 @@
+
+#### Rendering Logs
+
+The logs are rendered using the [`log-view`](https://github.com/mozilla/extension-activity-monitor/blob/master/src/lib/web-component/log-view/) web component. Any new log that doesn't match the filters will be hidden.
+
+#### Filtering Log Entries
+
+Individual log items can be hidden by user-defined filters. These filters are stored in the **Model** of the Activity Log page. A JSDoc explaining the filter object can be found [here](https://github.com/mozilla/extension-activity-monitor/blob/68d51940f1db397a0972658622bbdd39041436a7/src/lib/ext-activitylog.js#L21-L38). The **Model** is updated when the controller is notified of filter changes via [`onFilterChanged`](https://github.com/mozilla/extension-activity-monitor/blob/68d51940f1db397a0972658622bbdd39041436a7/src/lib/ext-activitylog.js#L338-L347). Ultimately, the [`log-view` hides](https://github.com/mozilla/extension-activity-monitor/blob/68d51940f1db397a0972658622bbdd39041436a7/src/lib/web-component/log-view/log-view-element.js#L28-L34) the table rows that didn't match the applied filters. The filters can be changed by the user via the following UI components:
+
+- [`filter-option`](https://github.com/mozilla/extension-activity-monitor/tree/master/src/lib/web-component/filter-option) provides the UI to filter by individual properties of log entries (such as extension id, view type, API name, API type). The available filter values are derived from the logs collected so far. During live logging, new unknown filter values may appear and be appended to the filter UI.
+- [`filter-keyword`](https://github.com/mozilla/extension-activity-monitor/tree/master/src/lib/web-component/filter-keyword) provides the UI to filter by substring. It searches the provided substring inside the `data` property of the log object.
+- [`filter-timestamp`](https://github.com/mozilla/extension-activity-monitor/tree/master/src/lib/web-component/filter-timestamp) provides the UI to filter by timestamps. The timestamps filter can be applied by choosing "start time" or "stop time" or both (from the context menu) to hide the logs that don't fall under the chosen timestamp range.
+
+When Activity Log page is opened through devtools panel, logs will be filtered by that tab's id. This filter cannot be changed through the UI.
+
+## Picking an issue
+
+If you are a new contributor looking for a easy bug/issue to get started, take a look at the list of [good first bugs](https://github.com/mozilla/extension-activity-monitor/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+bug"). Feel free to ask permission from the maintainers by commenting on any issue that you would like to work on.
+
+## Installation
+
+To install Extension Activity Monitor, follow the given instructions [here](https://github.com/mozilla/extension-activity-monitor/blob/master/README.md#installation).
+
+## Writing & Running Tests
+
+Unit test is done with [JEST Framework](https://jestjs.io/) in this project.
```

Comment:
> Unit test is done with [Jest Framework](https://jestjs.io/) in this project.
should be replaced 

Replace the intro with the following (partially merged with the next section).

> Tests are stored in the [tests directory](https://github.com/mozilla/extension-activity-monitor/tree/master/tests) and run with the [Jest framework](https://jestjs.io/).

Reply:
done :-)